content/en/docs/concepts/instrumentation/libraries.md

```diff
@@ -210,41 +204,41 @@ through corresponding client implementation.
 
 ![Nested database and HTTP spans in Jaeger UI](../nested-spans.svg)
 
-If OpenTelemetry does not support tracing your network client, use your best
-judgement, here are some considerations to help:
+If OpenTelemetry does not support tracing your network client, here are some
+considerations to help you decide the best course of action:
 
 - Would tracing network calls improve observability for users or your ability to
   support them?
 - Is your library a wrapper on top of public, documented RPC API? Would users
   need to get support from the underlying service in case of issues?
-  - instrument the library and make sure to trace individual network tries
+  - Instrument the library and make sure to trace individual network tries.
 - Would tracing those calls with spans be very verbose? or would it noticeably
   impact performance?
-  - use logs with verbosity or span events: logs can be correlated to parent
+  - Use logs with verbosity or span events: logs can be correlated to parent
     (public API calls), while span events should be set on public API span.
-  - if they have to be spans (to carry and propagate unique trace context), put
+  - If they have to be spans (to carry and propagate unique trace context), put
     them behind a configuration option and disable them by default.
 
 If OpenTelemetry already supports tracing your network calls, you probably don't
-want to duplicate it. There may be some exceptions:
+want to duplicate it. There might be some exceptions:
 
-- to support users without auto-instrumentation (which may not work in certain
-  environments or users may have concerns with monkey-patching)
-- to enable custom (legacy) correlation and context propagation protocols with
-  underlying service
-- enrich RPC spans with absolutely essential library/service-specific
-  information not covered by auto-instrumentation
+- To support users without auto-instrumentation, which might not work in certain
+  environments or when users have concerns with monkey-patching.
+- To enable custom or legacy correlation and context propagation protocols with
+  underlying service.
+- Enrich RPC spans with essential library or service-specific information not
+  covered by auto-instrumentation.
 
-WARNING: Generic solution to avoid duplication is under construction 🚧.
```

Comment:
@open-telemetry/docs-approvers Do we still need this? Not a big fan of "under construction" messages in docs. It's sort of implicit and can be verified by anyone interested.

Reply:
I'm find with removing the more warning-y text.